app/components/Edit/EditSchedule.js

```diff
@@ -98,6 +121,31 @@ class EditSchedule extends Component {
     return timeToTimeInputValue(time, true);
   }
 
+  toggle24Hours(day, is24Hours) {
```

Comment:
It'd be good to add a comment that briefly describes what `is24Hours` should be. Otherwise it's not clear without reading the code whether it represents the current value *before toggling* or the desired value *after toggling*.